.golangci.yml

```diff
@@ -31,6 +31,10 @@ linters:
       - wsl
       - exhaustivestruct
       - varnamelen
+      - maligned
+      - scopelint
+      - golint
+      - interfacer
```

Comment:
It will be nice to have comment describing why we disabled them, something like `// deprecated`

Reply:
Done